cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-detailed-ui.adoc

```diff
@@ -0,0 +1,290 @@
+Step 1 only needs to be performed the *first time* you are deploying into an AWS account. Step 2 only needs to be performed the *first time* you are using the user interface. For successive clusters of the same y-stream version, you only need to create the cluster.
+
+== Creating account wide roles
+
+[NOTE]
+====
+If you already have account roles from an earlier deployment, skip this step. The UI will detect your existing roles after you select an associated AWS account.
+====
+
+If this is the _first time_ you are deploying ROSA in this account and you have _not_ yet created the account roles, create the account-wide roles and policies, including the Operator policies.
+
+* In your terminal, run the following command to create the account-wide roles:
++
+[source,terminal]
+----
+rosa create account-roles --mode auto --yes
+----
++
+.Example output
++
+[source,terminal]
+----
+I: Creating roles using 'arn:aws:iam::000000000000:user/rosa-user'
+I: Created role 'ManagedOpenShift-ControlPlane-Role' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-ControlPlane-Role'
+I: Created role 'ManagedOpenShift-Worker-Role' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-Worker-Role'
+I: Created role 'ManagedOpenShift-Support-Role' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-Support-Role'
+I: Created role 'ManagedOpenShift-Installer-Role' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-Installer-Role'
+I: Created policy with ARN 'arn:aws:iam::000000000000:policy/ManagedOpenShift-openshift-machine-api-aws-cloud-credentials'
+I: Created policy with ARN 'arn:aws:iam::000000000000:policy/ManagedOpenShift-openshift-cloud-credential-operator-cloud-crede'
+I: Created policy with ARN 'arn:aws:iam::000000000000:policy/ManagedOpenShift-openshift-image-registry-installer-cloud-creden'
+I: Created policy with ARN 'arn:aws:iam::000000000000:policy/ManagedOpenShift-openshift-ingress-operator-cloud-credentials'
+I: Created policy with ARN 'arn:aws:iam::000000000000:policy/ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credent'
+I: To create a cluster with these roles, run the following command:
+rosa create cluster --sts
+----
+
+== Associating your AWS account with your Red Hat account
+This step tells the OCM what AWS account you want to use when deploying ROSA.
+
+[NOTE]
+====
+If you have already associated your AWS accounts, skip this step.
+====
+
+. Open the OCM by visiting the Red Hat link:https://console.redhat.com/openshift[console] and logging in to your Red Hat account.
+
+. Click on *Create Cluster*.
+
+. Scroll down to the {product-title} (ROSA) row and click on *Create Cluster*.
++
+image::cloud-experts-getting-started-rosa-deployment-detailed-ui-create.png[]
+
+. Check the box stating that you have read and completed all the prerequisites.
+
+. Click the dropbox under *Associated AWS account*. There might not be any associated accounts. This is expected since we have not yet associated any AWS accounts. 
```

Comment:
dropbox has been renamed to "Associated AWS infrastructure account * from ROSA Wizards.